vendor/gradle-plugin/build.gradle

```diff
@@ -46,20 +46,8 @@ android {
 }
 
 dependencies {
-    def supportVer = "28.0.0"
-    if (project.hasProperty("supportVersion")) {
-        supportVer = supportVersion
-    }
-    compileOnly "com.android.support:support-v4:$supportVer"
-    compileOnly "com.android.support:appcompat-v7:$supportVer"
-
-    configurations.all {
-       resolutionStrategy.eachDependency { DependencyResolveDetails details ->
-           if (details.requested.group == "com.android.support" && !details.requested.name.startsWith("multidex")) {
-               details.useVersion supportVer
-           }
-       }
-    }
```

Comment:
Should we keep this for  `androidX` too? In case a dep specifies a different androidX version, we likely want to use the same version across the board?

Reply:
The thing is all androidx packages have different versioning rules. That s why i removed it

Reply:
Ok, if that has no side-effects then that should be fine. I'm a bit worried it may lead to larger apk/aab's if we include multiple versions of the same lib. Would it make more sense to use the most recent of them all? (the one coming from the main gradle file)

Reply:
I dont think so. You cant use multiple versions in the apk (i dont think you can). Gradle will do the job